website/content/docs/lambda/registration/automate.mdx

```diff
@@ -0,0 +1,190 @@
+---
+layout: docs
+page_title: Automate Lambda Function Registeration
+description: >-
+  Register AWS Lambda functions with Consul service mesh using the Consul Lambda registrator. The Consul Lambda registrator automates Lambda function registration.
+---
+
+# Automate Lambda Function Registeration
+
+This topic describes how to automate Lambda function registration using the Consul Lambda registrator module for Terraform.
+
+## Introduction
+
+You can deploy the Lambda registrator to your environment to automatically register and deregister Lambda functions with Consul based on the function's tags. Refer to the [AWS Lambda tags documentation](https://docs.aws.amazon.com/lambda/latest/dg/configuration-tags.html) to learn about tags. 
+
+The registrator also stores and periodically updates information required to make mTLS requests to upstream services in the AWS parameter store. This enables Lambda functions to invoke mesh services. Refer to [Invoke Services from Lambda Functions](TODO) for additional information. 
+
+The registrator runs as a Lambda function that is invoked by AWS EventBridge. Refer to the [AWS EventBridge documentation](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-what-is.html) for additional information.
+
+EventBridge invokes the registrator using either [AWS CloudTrail](https://docs.aws.amazon.com/lambda/latest/dg/logging-using-cloudtrail.html) to synchronize with Consul in real-time or in scheduled intervals.
+
+CloudTrail events typically synchronize updates, registration, and deregistration within one minute, but events may occasionally be delayed.
+
+Scheduled events fully synchronize functions between Lambda and Consul to prevent entropy. By default, EventBridge triggers a full sync every five minutes.
+
+The following diagram shows the flow of events from EventBridge into Consul:
+
+
+<ImageConfig width={500}>
+
+![Lambda Registrator Architecture](/img/lambda_registrator_architecture.svg)
+
+</ImageConfig>
+
+1. EventBridge invokes the Lambda registrator based on CloudTrail Lambda events or a schedule.
+1. Lambda registrator determines how to reconcile Lambda's control plane state
+   with Consul state and ensures they are in sync by registering, updating, and
+   deregistering Lambda services.
+
+## Requirements 
+
+Verify that your environment meets the requirements specified in [Lambda Function Registration Requirements](/docs/lambda/registration/index).
+
+## Configuration 
+
+The Lambda registrator module stores data in the AWS parameter store. You can configure the type of data stored and how to store it. 
```

Comment:
```suggestion
The Lambda registrator module stores data in the AWS parameter store. You can configure the type of data stored and how to store it. 
```

What is module referring to here?

Reply:
Isn't it a module you add to the TF file? This description hasn't changed.